plugins/GSdx/GSDevice.cpp

```diff
@@ -133,9 +133,9 @@ void GSDevice::Present(GSTexture* sTex, GSTexture* dTex, const GSVector4& dRect,
 
 GSTexture* GSDevice::FetchSurface(int type, int w, int h, bool msaa, int format)
 {
-	GSVector2i size(w, h);
+	const GSVector2i size(w, h);
 
-	for(list<GSTexture*>::iterator i = m_pool.begin(); i != m_pool.end(); ++i)
+	for(std::deque<GSTexture*>::const_iterator i = m_pool.cbegin(); i != m_pool.cend(); ++i)
```

Comment:
I don't remember the performance impact of this code. But I think ideally, m_pool shall be split into an array of pool. The rational is that type/format/msaa take few value. Beside I think it would be nice to have a different profile based on m_type.

RenderTarget are often quite expensive in memory, so we don't want too many of them. Whereas texture are smaller so it make sense too keep more pool for them.

Finally, a smaller pool would allow to use a basic vector.

I'm fine with current code. It was just a general comment.